.pre-commit-config.yaml

```diff
@@ -188,7 +188,8 @@ repos:
     name: MyPy, for Python 3.13
     additional_dependencies:
     - lxml  # dep of `--txt-report`, `--cobertura-xml-report` & `--html-report`
-    - pytest
+    - pytest  # dep of `credential_plugins_test.py`& `importable_test.py`
```

Comment:
I think that `pytest` does not need explanation since it's a commonly used framework that will be in each test file. It'd be silly to enumerate like 50 test modules when the tests grow. Either don't explain `pytest` at all or just summarize that it's used in tests and MyPy lints those files too.

Reply:
Marking this thread as resolved.